### PR TITLE
zh-CN: translate unsafe

### PR DESCRIPTION
**Pull request by mgeisler**
Part of #324.

### CODE REVIEW

File: po/zh-CN.po [hunk 1]
```diff
@@ -11138,6 +11190,9 @@ msgid ""
 "undefined behaviour under any\n"
 "arbitrary circumstances."
 msgstr ""
+"这个问题通常仅存在于使用指针执行违反 Rust 内存模型的操作的外部函数中。\n"
+"但一般而言，任何有可能在任意情况下出现未定义行为的\n"
+"C 函数，都有可能存在这个问题。"
```
Comment (superwhd) [inline po/zh-CN.po:11195]:
```suggestion
"但一般而言，任何 C 函数都有可能在任意情况下出现未定义行为。"
```

File: po/zh-CN.po [hunk 2]
```diff
@@ -10797,36 +10797,42 @@ msgstr ""
...
+"本课程中出现的大多为“安全 Rust”，但是了解“不安全 Rust”的定义\n"
+"非常重要。"
 
 #: src/unsafe.md:11
 msgid ""
 "Unsafe code is usually small and isolated, and its correctness should be "
 "carefully\n"
 "documented. It is usually wrapped in a safe abstraction layer."
 msgstr ""
+"不安全的代码通常内容很少而且隔离运行，\n"
```
[the remark this answers is not in the document]
Comment (mgeisler) [reply po/zh-CN.po:10830]: > My understanding of 'isolated' here is that it's isolated from other code, which is not about execution.

Yes, you're spot on!

**Review by superwhd — APPROVED** [review]
LGTM 👍

File: po/zh-CN.po (hunk 2 above)
[the remark this answers is not in the document]
Comment (mgeisler) [reply po/zh-CN.po:10830]: @superwhd and @wnghl, do you want to add a suggest here before we merge this?

Comment (superwhd) [inline po/zh-CN.po:10830]:
```suggestion
"不安全的代码通常内容很少而且与其他代码隔离，\n"
```

[the remark this answers is not in the document]
Comment (mgeisler) [reply po/zh-CN.po]: Thanks for adding the suggestion!

**Pull request merged**